testing/releases/latest.go

```diff
@@ -0,0 +1,55 @@
+package releases
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/circleci/ex/testing/download"
+)
+
+// DownloadLatest is a helper that will download the latest which from the release server at baseURL
+func DownloadLatest(ctx context.Context, baseURL, which, pinned, dir string) (string, error) {
```

Comment:
Can we make this take a struct, so we don't have breakages if we add more args

Reply:
already added one more param :p